price-estimator/src/main.rs

```diff
@@ -51,6 +55,8 @@ struct Options {
     orderbook_update_interval: Duration,
 }
 
+type Orderbook = orderbook::Orderbook<EventBasedOrderbook>;
+
 fn main() {
```

Comment:
Maybe this should be `async main` now (because we have futures in the main method now)? You could use `tokio::main` macro for that.

Reply:
As you mentioned below, the reason I didn't use tokio::main is that we configure the runtim. I wasn't sure how tokio configures it automatically when don't do that. At the moment I prefer being explicit in this case.